src/glog/logging.h.in

```diff
@@ -86,6 +86,22 @@
 #include <gflags/gflags.h>
 #endif
 
+// Check if it's compiled in C++11 mode.
+//
+// GXX_EXPERIMENTAL_CXX0X is defined by gcc and clang up to at least
+// gcc-4.7 and clang-3.1 (2011-12-13).  __cplusplus was defined to 1
+// in gcc before 4.7 (Crosstool 16) and clang before 3.1, but is
+// defined according to the language version in effect thereafter.
+// Microsoft Visual Studio 14 (2015) sets __cplusplus==199711 despite
+// reasonably good C++11 support, so we set LANG_CXX for it and
+// newer versions (_MSC_VER >= 1900).
+#define SUPPORT_CPLUSPLUS11 (defined(__GXX_EXPERIMENTAL_CXX0X__) || __cplusplus >= 201103L || \
```

Comment:
`std::atomic<>` availability should be checked in `CMakeLists.txt` using `CheckCXXSymbolExists` or `CheckCXXSourceCompiles` module.

Reply:
Will do!